frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -170,6 +174,8 @@ const loadPriorityMap: Record<TileId, number> = {
     [TileId.PAGE_REPORTS_TIMEZONES]: 13,
     [TileId.PAGE_REPORTS_LANGUAGES]: 14,
     [TileId.PAGE_REPORTS_TOP_EVENTS]: 15,
+    [TileId.MARKETING]: 16,
+    [TileId.MARKETING_CAMPAIGN_BREAKDOWN]: 17,
```

Comment:
I guess we can break this down by tab now, they only matter for on-screen concurrency. No need to change it, just commenting.